mxcubecore/HardwareObjects/abstract/AbstractAperture.py

```diff
@@ -53,21 +53,24 @@ def init(self):
             logging.getLogger("HWR").error("Aperture: no position list defined")
 
     def get_diameter_size_list(self):
-        """
+        """Get diameters size list
```

Comment:
Might as well be a bit more verbose:

```suggestion
        """Get list of diameter sizes.
```